src/test/java/gov/healthit/chpl/aqa/stepDefinitions/LoginSteps.java

```diff
@@ -54,10 +62,26 @@ public void logInAsAcb() {
 
     /**
      * Verify login attempt was successful.
+     * @param role as ACB or Admin
      */
-    @Then("^I should be logged in to CHPL as ACB Admin$")
-    public void verifyLoginWasSuccessful() {
+    @Then("^I should be logged in to CHPL as \"([^\"]*)\"$")
+    public void verifyLoginWasSuccessful(final String role) {
         String actualString = LoginPage.welcomeText(driver).getText();
         assertTrue(actualString.contains("Welcome"));
     }
+
+    /**
+     * Log the user in as an ACB.
+     */
+    @Given("^I'm logged in as an Admin$")
+    public void logInAsAdmin() {
+        driver.get(url + "#/admin/dpManagement");
+        WebDriverWait wait = new WebDriverWait(driver, TIMEOUT);
+        wait.until(ExpectedConditions.visibilityOf(LoginPage.username(driver)));
+        LoginPage.username(driver).sendKeys(adminUsername);
```

Comment:
Here, do a switch or if/else based on the ROLE string to figure out what username and password you should use:
```
String username;
String password
if (role.equalsIgnoreCase("ROLE_ADMIN")) {
username = roleAdminUsername
... etc.
```